scripts/install/functions.sh

```diff
@@ -85,43 +85,44 @@ function create_shell_app {
 
 function add_b2b {
     if [ "${ADD_B2B_LIBS}" = true ] ; then
-        ng add --skip-confirmation @spartacus/organization@${SPARTACUS_VERSION} --interactive false
-        ng add --skip-confirmation @spartacus/checkout@${SPARTACUS_VERSION} --interactive false --features="Checkout-B2B" --features="Checkout-Scheduled-Replenishment"
+        ng add @spartacus/organization@${SPARTACUS_VERSION} --skip-confirmation --no-interactive
```

Comment:
Overall, we might need to update the spartacus schematics command line examples throughout the documentation.  Perhaps we need to make a full text search to catch instances that use parameters we need to change.